runtime/neumann/src/lib.rs

```diff
@@ -164,6 +158,8 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	state_version: 0,
 };
 
+pub const NATIVE_TOKEN_ID: TokenId = 0;
```

Comment:
you need to use this in the gensis config to make this true right?

Reply:
You can’t add assets in the genesis config.